### ISSUE TITLE
Error in redux-saga/yield-effects

### ISSUE THREAD

**Issue by nktssh** [pke/eslint-plugin-redux-saga#12]
For plugin v0.2.0 
and code like this:
```
import { put } from 'redux-saga/effects';

function* exampleSaga() {
  // ... some code
  yield [
        put(someActionCreator()),
        put(anotherActionCreator()),
  ];
}
```
get following linting error:
`error`  `put effect must be yielded`  `redux-saga/yield-effects`

**Comment by nktssh** [comment]
the same code gives no errors in `v.0.1.5`

**Comment by pke** [comment]
Yes, arrays are not yet handled :( Will see when I can work on that. Could you help out @burabure?

**Comment by pke** [comment]
@nktssh 
> the same code gives no errors in v.0.1.5
That's because this version did not detect effects imported via `redux-saga/effects`

**Comment by burabure** [comment]
this should be easy to fix. If i get a minute I'll submit a PR; let's see who gets to it first @pke
